### ISSUE TITLE
Add ability to get response code value from request response

### ISSUE THREAD

**Issue by Burbaka** [stanfy/helium#74]
For the following example Helium allows only to check if response code equals some hardcoded values. But I would like to get exact number with was received.

```groovy
scenario "create response test" spec {   
   def requestResponse = post 'some/endpoint/@device_id/create' with {
        path {
          device_id deviceId
        }
      }

      // TODO check for custom response code
      // requestResponse.mustBeClientError()
      // requestResponse.mustBeClientError() 
}

```

**Comment by roman-mazur** [comment]
Duplicated by #83. Fixed in 63043ee8b84a1b8943138ecb0eaea1f6aa6e9228.

**Comment by roman-mazur** [comment]
Available in the latest snapshot.
